projects/plugins/jetpack/extensions/blocks/blog-stats/blog-stats.php

```diff
@@ -47,7 +47,25 @@ function register_block() {
 function load_assets( $attributes ) {
 	Jetpack_Gutenberg::load_assets_as_required( __DIR__ );
 
-	$stats = 0;
+	// For when Stats has been disabled subsequent to inserting the block.
+	if ( ! \Jetpack::is_module_active( 'stats' ) ) {
+		if ( current_user_can( 'edit_theme_options' ) ) {
+			return sprintf(
+				'<p>%s</p>',
+				sprintf(
+					/* translators: placeholder is a link that says "Stats module". */
+					esc_html__( 'Please enable the %s in Jetpack to use this block.', 'jetpack' ),
+					'<a href="' .
+						esc_url( admin_url( 'admin.php?page=jetpack_modules&module_tag=Jetpack%20Stats' ) ) .
+						'">' .
+						esc_html__( 'Stats module', 'jetpack' ) .
```

Comment:
Just saying "stats" would feel more natural and less technical, than "module". :-)

Reply:
Agreed - I’ll update the string! I also think it’s better to have the link on “enable” too since that’s the action verb here.

Reply:
https://github.com/Automattic/jetpack/pull/36127 for this